backend/RepositoryLayer/EventRepository.cs

```diff
@@ -41,29 +41,23 @@ public interface IEventRepository : IBaseRepository<Event>
         /// <returns>true wenn korrekt angepasst, ansonsten false</returns>
         Task<bool> ModifyEventAsync(EventDto eventToModify);
 
-        /// <summary>
-        /// Hinzufügen eines eventContacts
-        /// </summary>
-        /// <param name="eventContact">das eventContact</param>
-        /// <returns>das erzeugte EventContact</returns>
         Task<EventContact> AddEventContactAsync(EventContact eventContact);
 
-        /// <summary>

```

Comment:
Warum werden die Kommentare gelöscht?

Reply:
Ich habe ungenutzte Funktionen komplett rausgelöscht. Einschließlich deren Kommentare :) die hatte ich mal in der Vergangenheit angelegt, in dem Glauben man könne sie gebrauchen.. War dann doch nicht so